proxy/src/http/org.rs

```diff
@@ -17,10 +17,13 @@ pub fn filters(
     subscriptions: notification::Subscriptions,
 ) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
     get_filter(Arc::<RwLock<registry::Registry>>::clone(&registry))
+        .or(list_filter(Arc::<RwLock<registry::Registry>>::clone(
```

Comment:
FYI you don't need to specify the full type here (and in other places), you can do `Arc::clone(&registry)`

Reply:
Thanks! Replaced it.